index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Product Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <body>
+      <div id="page-wrapper">
+        <header id="header">
+          <div class="logo">
+            <img
+              id="header-img"
+              src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+              
+            />
```

Comment:
Please add a link for the logo. In the future, your logo will be clickable and remove extra white spaces.

Reply:
- [ ] Nice communicating with the student but please point out which kind of link the student should use for the logo to be clickable and direct us to the desired section. try to be more specific as much as possible. you can give more details to the student for example.
**it would be better to wrap your logo with an` a tag ` . an `a tag` gives you this ability to let you connect and link with other sections.**

Reply:
- [ ] **Congratulations on implementing some of the previous required changes** :1st_place_medal: :1st_place_medal: But as suggested by the previous Code Reviewer, I think it would be better to be more descriptive in this comment by providing **WHY** with more details and **HOW** to help the code writer and it would be better too to give some examples of what you are suggesting :+1: